draft-ietf-taps-arch.md

```diff
@@ -97,7 +105,13 @@ This document was developed in parallel with the specification of the Transport
 
 ## Background
 
-The Transport Services architecture is based on the survey of services provided by IETF transport protocols and congestion control mechanisms {{?RFC8095}}, and the distilled minimal set of the features offered by transport protocols {{?I-D.ietf-taps-minset}}. These documents identified common features and patterns across all transport protocols developed thus far in the IETF.
+The BSD Unix Sockets API's (Sockets) and its SOCK_STREAM abstraction was a revolution in simplicity: by bringing network sockets into the UNIX programming model, it enabled anyone who knew how to write programs that dealt with sequential-access files to also write network applications. The simplicity of this API is a key reason the	Internet won the protocol wars {{PROTOCOL-WARS}} of the 1980s. SOCK_STREAM is	tied to the Transmission Control Protocol (TCP), specified in 1981 {{?RFC0793}}.	
```

Comment:
Why the brackets around "(Sockets)"? Without the word, the sentence wouldn't work anymore.
Also, I think we call it "Socket API", not "Sockets API" in the rest of the document.